witnet.toml

```diff
@@ -10,40 +10,40 @@ public_addr = "0.0.0.0:21337"
 # them for a list of all the addresses they know. From that moment on, the addresses listed here are not given any
 # preference over addresses that have been discovered in any way.
 known_peers = [
-        "5.189.144.203:21337",
         "52.166.178.145:21337",
-        "62.171.170.204:22337",
+        "62.171.170.204:21337",
         "64.225.67.5:21337",
         "64.227.34.207:21337",
-        "82.213.202.243:23337",
+        "68.183.204.241:21337",
         "95.111.247.115:21337",
-        "95.217.50.200:21337",
         "116.203.234.252:21337",
         "128.199.91.152:21337",
         "135.181.36.168:21337",
         "138.201.155.65:21337",
         "139.162.102.104:21337",
+        "142.93.250.57:21337",
         "144.91.104.168:21337",
         "157.245.67.41:21337",
         "159.69.146.71:21337",
         "161.35.172.136:21337",
         "161.97.64.253:21337",
         "164.90.183.45:21337",
-        "165.227.51.79:21337",
         "167.71.210.157:21337",
+        "167.86.109.249:21337",
+        "167.86.112.22:21337",
         "167.86.112.23:21337",
-        "167.172.196.106:21337",
-        "172.104.233.132:21337",
-        "172.105.68.149:21337",
-        "173.212.245.133:21337",
-        "178.62.85.63:21337",
+        "167.99.154.252:21337",
+        "173.249.3.178:21337",
+        "173.249.8.65:213377",
```

Comment:
This is a typo. :)

Also, that VPS will run the superblock bootstrap committee node, not sure if you want to have it do both since I have seen nodes crash because of too many incoming connections.

Reply:
Thanks!! I would recommend you to run the superblock bootstrap committee node in a machine that you know is not going to crash, for example in a machine with no inbounds, and respect to that IP, if you detect problems with inbound connection you can add an inbound_limit in the witnet.toml file